app/lib/three_scale/redis_config.rb

```diff
@@ -28,7 +28,6 @@ def db
     end
 
     def prone_to_key_collision_with?(other)
```

Comment:
I haven't found any reference to this method in the code. It could be dead code.

Reply:
Yep, I guess it was last used here: https://github.com/3scale/porta/pull/2687/files#diff-6f3bfe3c6f2ddae879fd71e02b86b3bcc43a08af77b1354a6604e7dc66588ae7L44

I'll remove also.